src/io/StringConverter.cpp

```diff
@@ -115,7 +115,7 @@ StringConverter::Convert(char *narrow)
     gcc_fallthrough;
 
   case Charset::AUTO:
-    return narrow;
+    return const_cast<TCHAR*>(narrow);
```

Comment:
This is bad.

Reply:
Ok, I will use the previous component without any changes